docs/CONFIG.md

```diff
@@ -7516,6 +7518,12 @@ TipCapMinimum is the minimum gas tip to use when submitting transactions to the
 
 (Only applies to EIP-1559 transactions)
 
+### CostMax
+```toml
+CostMax = '1000000000000000000 wei' # 1 ETH in wei # Default
+```
+CostMax is the limit gas cost
```

Comment:
nit: CostMax is the maximum gas cost allowed for a single Tx.

Reply:
done